bin/Powheg/production/pre2017/13TeV/dijet_CT14NLO_13TeV/dijet_CT14NLO_13TeV_powheg.input

```diff
@@ -0,0 +1,91 @@
+
+! A typical call uses 1/1400 seconds (1400 calls per second)
+ncall1 20000         ! No. calls for the construction of the importance sampling grid
+itmx1 5              ! No. iterations for grid: total 100000 calls ~ 70 seconds
+ncall2 20000         ! No. calls for the computation of the upper bounding
+                     ! envelope for the generation of radiation
+itmx2 5              ! No. iterations for the above
+
+! Notice: the total number of calls is ncall2*itmx2*foldcsi*foldy*foldphi
+! these folding numbers yield a negative fraction of 0.5% with bornktmin=10 GeV.
+! With these settings: ncall2*itmx2*foldcsi*foldy*foldphi=5M, 60 minutes
+foldcsi 5            ! No. folds on csi integration
+foldy   10            ! No. folds on  y  integration
+foldphi 2            ! No. folds on phi integration
+
+nubound 1000000       ! No. calls to set up the upper bounding norms for radiation.
+                     ! This is performed using only the Born cross section (fast)
+
+! OPTIONAL PARAMETERS
+
+
+#bornonly 1          ! (default 0). If 1 compute underlying Born using LO
+                     ! cross section only.
+
+#ptsqmin    0.8      ! (default 0.8 GeV) minimum pt for generation of radiation
+#charmthr   1.5      ! (default 1.5 GeV) charm treshold for gluon splitting 
+#bottomthr  5.0      ! (default 5.0 GeV) bottom treshold for gluon splitting
+#testplots  1        ! (default 0, do not) do NLO and PWHG distributions
+#charmthrpdf  1.5    ! (default 1.5 GeV) pdf charm treshold
+#bottomthrpdf 5.0    ! (default 5.0 GeV) pdf bottom treshold
+
+#xupbound 2d0        ! increase upper bound for radiation generation
+
+iseed    SEED       ! Start the random number generator with seed iseed
+#rand1     0         ! skipping  rand2*100000000+rand1 numbers (see RM48
+#rand2     0         ! short writeup in CERNLIB).              
+#manyseeds 1         ! Used to perform multiple runs with different random
+                     ! seeds in the same directory.
+                     ! If set to 1, the program asks for an integer j;
+                     ! The file pwgseeds.dat at line j is read, and the
+                     ! integer at line j is used to initialize the random
+                     ! sequence for the generation of the event.
+                     ! The event file is called pwgevents-'j'.lhe
+
+doublefsr 1          ! Default 0; if 1 use new mechanism to generate regions
+                     ! such that the emitted harder than the
+                     ! emitter in FSR is suppressed. If doublefsr=0 this is
+                     ! only the case for emitted gluons (old behaviour). If
+                     ! 1 it is also applied to emitted quarks.
+                     ! If set, it strongly reduces spikes on showered output.
+
+
+
+#par_diexp 4         ! default is 2. With 4 there is a stronger separation
+#par_dijexp 4        ! of regions, it may help to reduce spikes when generating
```

Comment:
there are many commented lines, it would be good to remove them for  better clarity

Reply:
I will remove the commented lines.